src/components/network/network-panel.js

```diff
@@ -111,6 +120,40 @@ genotet.NetworkPanel.prototype.edgeListContainer = function() {
 };
 
 /**
+ * Updates the network list for panel after loading network list.
+ */
+genotet.NetworkPanel.prototype.updateFileListAfterLoading = function() {
+  if (!this.fileSelectIsOpen_) {
```

Comment:
ditto. Please try to a void an entire block of code nested on a single condition.